### PR TITLE
add eventually to test to be more resilient to slow google group propagation

### PR DESCRIPTION
**Pull request by andy7i**
\<your comments for this PR go here\>

Have you read [CONTRIBUTING.md](../CONTRIBUTING.md) lately? If not, do that first.

I, the developer opening this PR, do solemnly pinky swear that:

- [ ] I've followed [the instructions](https://github.com/broadinstitute/firecloud-orchestration/blob/develop/CONTRIBUTING.md#api-changes) if I've made any changes to the API, _especially_ if they're breaking changes
- [ ] I've updated the RC_XXX release ticket with any manual steps required to release this change
- [ ] I've updated the [FISMA documentation](https://github.com/broadinstitute/firecloud-orchestration/blob/develop/CONTRIBUTING.md#fisma-documentation-changes) if I've made any security-related changes, including auth, encryption, or auditing

In all cases:

- [ ] Get two thumbsworth of review and PO signoff if necessary
- [ ] Verify all tests go green
- [ ] Squash and merge; you can delete your branch after this **unless it's for a hotfix**. In that case, don't delete it!
- [ ] Test this change deployed correctly and works on dev environment after deployment

---

very unsatisfactory change but it may help with the issues we're facing right now to unblock the release

### CODE REVIEW

**Comment by andy7i** [comment]
running repeated tests here: https://fc-jenkins.dsp-techops.broadinstitute.org/job/swatomation-pipeline-run-hourly-testing-qa-1534/

**Review by s-rubenstein — APPROVED** [review]
LGTM. Is there a ticket for the longer term fix?

**Review by gpolumbo-broad — APPROVED** [review]
Approved, but this isn't something we should leave around if we can avoid it.  Might consider adding another comment to try removing this line of code and/or a ticket for us to follow up on this line to remove it later?  It's not the end of the world if it hangs around, but would be best if we remember to clean it up at some point.

File: automation/src/test/scala/org/broadinstitute/dsde/test/api/orch/WorkspaceApiSpec.scala [hunk 1]
```diff
@@ -39,6 +39,8 @@ class WorkspaceApiSpec extends FreeSpec with Matchers
           withWorkspace(projectName, prependUUID("writer-storage-cost"), aclEntries = List(AclEntry(writer.email, WorkspaceAccessLevel.Writer))) { workspaceName =>
             Orchestration.workspaces.waitForBucketReadAccess(projectName, workspaceName)(ownerAuthToken)
 
+            Thread.sleep(60 * 1000) // This was added as a workaround for https://broadworkbench.atlassian.net/browse/QA-1534
```
Comment (MatthewBemis) [inline automation/src/test/scala/org/broadinstitute/dsde/test/api/orch/WorkspaceApiSpec.scala]: `eventually` may be more appropriate to use here. You can tweak the timeout for it using the implicit `PatienceConfig` value

`OrchestrationApiSpec.scala` has some precedent for this, if you're curious

**Comment by andy7i** [comment]
@gpolumbo-broad @s-rubenstein I made the change @MatthewBemis suggested which seems reasonable even as a longer-term fix, so I didn't add a follow-up / cleanup ticket.

**Comment by andy7i** [comment]
`Scala CI / build (push) ` test failed, but looks unrelated:

```
[info] - should delete a published workspace successfully
Error:  [14:06:23.690] [org-broadinstitute-dsde-firecloud-service-WorkspaceServiceSpec-akka.actor.default-dispatcher-6] o.b.d.f.service.WorkspaceService - Unable to update this workspace, unpublishfailure:name, to false in elastic search.
org.broadinstitute.dsde.firecloud.FireCloudException: Failed to remove document with id unpublishfailure from elastic search
	at org.broadinstitute.dsde.firecloud.service.MockSearchDeleteWSDAO.deleteDocument(WorkspaceServiceSpec.scala:137)
	at org.broadinstitute.dsde.firecloud.service.WorkspacePublishingSupport.removeDocument(WorkspacePublishingSupport.scala:26)
	at org.broadinstitute.dsde.firecloud.service.WorkspacePublishingSupport.removeDocument$(WorkspacePublishingSupport.scala:25)
	at org.broadinstitute.dsde.firecloud.service.WorkspaceService.removeDocument(WorkspaceService.scala:32)
	at org.broadinstitute.dsde.firecloud.service.WorkspacePublishingSupport.$anonfun$setWorkspacePublishedStatus$2(WorkspacePublishingSupport.scala:52)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
	at scala.concurrent.Future$.$anonfun$apply$1(Future.scala:659)
	at scala.util.Success.$anonfun$map$1(Try.scala:255)
	at scala.util.Success.map(Try.scala:213)
	at scala.concurrent.Future.$anonfun$map$1(Future.scala:292)
	at scala.concurrent.impl.Promise.liftedTree1$1(Promise.scala:33)
	at scala.concurrent.impl.Promise.$anonfun$transform$1(Promise.scala:33)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:64)
	at akka.dispatch.BatchingExecutor$AbstractBatch.processBatch(BatchingExecutor.scala:55)
	at akka.dispatch.BatchingExecutor$BlockableBatch.$anonfun$run$1(BatchingExecutor.scala:92)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
	at scala.concurrent.BlockContext$.withBlockContext(BlockContext.scala:85)
	at akka.dispatch.BatchingExecutor$BlockableBatch.run(BatchingExecutor.scala:92)
	at akka.dispatch.TaskInvocation.run(AbstractDispatcher.scala:41)
	at akka.dispatch.ForkJoinExecutorConfigurator$AkkaForkJoinTask.exec(ForkJoinExecutorConfigurator.scala:49)
	at akka.dispatch.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260)
	at akka.dispatch.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339)
	at akka.dispatch.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979)
	at akka.dispatch.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107)
```
https://github.com/broadinstitute/firecloud-orchestration/runs/3202670744

**Comment by andy7i** [comment]
unrelated test failures in `Scala CI / build (pull_request) `
```
[info] delete workspace
[info] - should delete an unpublished workspace successfully
[info] - should delete a published workspace successfully
Error:  [14:08:30.349] [org-broadinstitute-dsde-firecloud-service-WorkspaceServiceSpec-akka.actor.default-dispatcher-7] o.b.d.f.service.WorkspaceService - Unable to update this workspace, unpublishfailure:name, to false in elastic search.
org.broadinstitute.dsde.firecloud.FireCloudException: Failed to remove document with id unpublishfailure from elastic search
	at org.broadinstitute.dsde.firecloud.service.MockSearchDeleteWSDAO.deleteDocument(WorkspaceServiceSpec.scala:137)
	at org.broadinstitute.dsde.firecloud.service.WorkspacePublishingSupport.removeDocument(WorkspacePublishingSupport.scala:26)
	at org.broadinstitute.dsde.firecloud.service.WorkspacePublishingSupport.removeDocument$(WorkspacePublishingSupport.scala:25)
	at org.broadinstitute.dsde.firecloud.service.WorkspaceService.removeDocument(WorkspaceService.scala:32)
	at org.broadinstitute.dsde.firecloud.service.WorkspacePublishingSupport.$anonfun$setWorkspacePublishedStatus$2(WorkspacePublishingSupport.scala:52)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
	at scala.concurrent.Future$.$anonfun$apply$1(Future.scala:659)
	at scala.util.Success.$anonfun$map$1(Try.scala:255)
	at scala.util.Success.map(Try.scala:213)
	at scala.concurrent.Future.$anonfun$map$1(Future.scala:292)
	at scala.concurrent.impl.Promise.liftedTree1$1(Promise.scala:33)
	at scala.concurrent.impl.Promise.$anonfun$transform$1(Promise.scala:33)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:64)
	at akka.dispatch.BatchingExecutor$AbstractBatch.processBatch(BatchingExecutor.scala:55)
	at akka.dispatch.BatchingExecutor$BlockableBatch.$anonfun$run$1(BatchingExecutor.scala:92)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
	at scala.concurrent.BlockContext$.withBlockContext(BlockContext.scala:85)
	at akka.dispatch.BatchingExecutor$BlockableBatch.run(BatchingExecutor.scala:92)
	at akka.dispatch.TaskInvocation.run(AbstractDispatcher.scala:41)
	at akka.dispatch.ForkJoinExecutorConfigurator$AkkaForkJoinTask.exec(ForkJoinExecutorConfigurator.scala:49)
	at akka.dispatch.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260)
	at akka.dispatch.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339)
	at akka.dispatch.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979)
	at akka.dispatch.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107)
```
https://github.com/broadinstitute/firecloud-orchestration/pull/866/checks?check_run_id=3202634546

**Comment by andy7i** [comment]
`Scala CI / build (pull_request)` failed for a second time, with a different test case this time

```
[info]   - should 400 if a bagit doesn't have either participants or samples *** FAILED ***
[info]     500 Internal Server Error did not equal 400 Bad Request (WorkspaceApiServiceSpec.scala:1019)
[info]     Analysis:
[info]     StatusCodes$ServerError(defaultMessage: "The[re was an internal server error]." -> "The[ request contains bad syntax or cannot be fulfilled].", intValue: 500 -> 400, reason: "[Internal Server Error]" -> "[Bad Request]")

```

https://github.com/broadinstitute/firecloud-orchestration/pull/866/checks?check_run_id=3202708896

**Review by MatthewBemis — APPROVED** [review]
LGTM!

**Pull request merged**